src/directives/ctr-input.ts

```diff
@@ -157,6 +158,19 @@ export class CtrInput {
         }
     }
 
+    @HostListener("click", ["$event"])
+    public onClick(event: any) {
+        this.el.nativeElement.select();
```

Comment:
sorry, what I meant was that there should be another property (or option) that let's the user to choose whether use `selectOnClick` or not